src/slack/slack-bot.js

```diff
@@ -206,6 +206,21 @@ export class SlackBot extends Bot {
       .then(() => Promise.delay(this.postMessageDelay));
   }
 
+  // Get the bot's name and a list of aliases suitable for use in a top-level
+  // command message handler.
+  getBotNameAndAliases() {
+    const {activeUserId, dataStore} = this.slack.rtmClient;
```

Comment:
I'm not wild about the fact that this tightly couples us to the slack.rtmClient. Could we push *some* of this into userland?